examples/lithops/gcf/README.md

```diff
@@ -0,0 +1,42 @@
+# Examples running Cubed on Lithops (Google Cloud Functions, GCS)
+
+## Pre-requisites
+
+1. A [GCP](https://cloud.google.com/) account, with [Functions](https://lithops-cloud.github.io/docs/source/compute_config/gcp_functions.html#installation) and [Storage](https://lithops-cloud.github.io/docs/source/storage_config/gcp_storage.html#installation) enabled.
+
+## Set up
+
+1. Install a Python environment with the basic package requirements:
+
+```shell
+# from this directory
+conda create --name cubed-lithops-gcf-examples -y python=3.8
+conda activate cubed-lithops-gcf-examples
+pip install -r requirements.txt  # use requirements file from same directory as this readme
+```
+
+2. Configure Lithops with a [Google Cloud Functions compute backend](https://lithops-cloud.github.io/docs/source/compute_config/gcp_functions.html#configuration), and a [Google Cloud Storage backend](https://lithops-cloud.github.io/docs/source/storage_config/gcp_storage.html#configuration).
+   - Note: it may be useful to put the configuration in a different place to the default (e.g. `~/.lithops/config.gcf`), and then call `export LITHOPS_CONFIG_FILE=~/.lithops/config.gcf`)
+3. Give permissions to access your GCP buckets by calling `export GOOGLE_APPLICATION_CREDENTIALS=<path-to-json>`, where `<path-to-json>` is the full path to the `.json` key file you downloaded when configuring GCP storage for Lithops in the previous step.
```

Comment:
I found I had to do this even when my lithops config file contained entries for `credentials_path`. If I don't do this step I get a permissions error from zarr array creation.

This means the `credentials_path` entry from the lithops config file is potentially redundant.

My understanding of why I have to do this is that cubed creates an empty zarr store in the bucket for temporary files upon initialization of the first `cubed.Array`, but at this point the lithops config has not yet been passed to the executor via `.compute()`, so gcsfs looks for the environment variable.

Reply:
(I wonder what would happen if our bucket for lithops config required a different set of permissions from the bucket we want to put temporary zarr data in... Perhaps then we would need both sets of permissions, and they would need to be two different permissions files.)